app/model/dashboard.model.js

```diff
@@ -81,6 +81,20 @@ function getSkillLevelForUser(userToSkillConnector, userId, skillId) {
     return level;
 }
 
+function getFutureSkillLevelForUser(userToSkillConnector, userId, skillId) {
+    var futureLevel = '';
+
+    for (var i = 0; i < userToSkillConnector.length; i++) {
```

Comment:
While you are in this file, could you change all the for-loops here to: 
 * Not store a variable for the intermediate return result.
 * Return in the loop instead of break.

i/e: 
function getFutureSkillLevelForUser(userToSkillConnector, userId, skillId) {
     for (var i = 0; i < userToSkillConnector.length; i++) {
        if (userId === userToSkillConnector[i].userId && skillId === userToSkillConnector[i].skillId) {
             return userToSkillConnector[i].futureLevel;
         }
     }
 }

would be much appreciated.